gamify-specs/src/main/resources/api-spec.yaml

```diff
@@ -59,6 +59,66 @@ paths:
         '201':
           description: created
 
+  /categories:
+    post:
+      description: post a new category
+      operationId: addCategory
+      requestBody:
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/Category'
+      responses:
+        '201':
+          description: created
+    get:
+      description: read a category
+      operationId: getCategory
+      parameters:
+        - name: idCategory
+          in: query
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: success
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Category'
```

Comment:
This endopoint should return all categories. To return only one category, we should have an endpoint of the form `/categories/myCategoryId`